test/tools/libtesteth/Options.cpp

```diff
@@ -48,7 +48,8 @@ void printHelp()
     cout << setw(35) << "-g <index>" << setw(25) << "Set the transaction gas array index when running GeneralStateTests\n";
     cout << setw(35) << "-v <index>" << setw(25) << "Set the transaction value array index when running GeneralStateTests\n";
     cout << setw(35) << "--singletest <TestName>" << "Run on a single test\n";
-    cout << setw(35) << "--singletest <TestFile> <TestName>" << "Run on a single test from file\n";
+    cout << setw(35) << "--singletest <TestFile>" << "Run tests from file. Require -t <TestSuite>\n";
+    cout << setw(35) << "--singletest <TestFile> <TestName>" << "Run on a single test from file. Require -t <TestSuite>\n";
```

Comment:
And this should probably better be `--testfile <TestFile> --singletest <TestName>`

Reply:
Actually, doesn't each file contain exactly one test always?

Reply:
It does now in the result files. I'm not sure if source files are relevant here. Having `--testfile` (or `--file`) and dropping `--singletest` would be great.

Reply:
source files not really relevant, the option should just run the filled file